<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>debugWindow</class>
 <widget class="QWidget" name="debugWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VSCAN DRIVER DEBUG WINDOW</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: rgb(85, 170, 255);</string>
  </property>
  <widget class="QFrame" name="mainFrame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>600</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-width: 1px;
border-style: solid; 
border-color: darkblue;
border-color: rgb(0, 0, 127);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="connectionFrame_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>531</width>
      <height>561</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width: 1px;
border-style: solid; 
border-color: darkblue;
border-color: rgb(0, 0, 127);
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPlainTextEdit" name="canText">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>511</width>
       <height>211</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
</string>
     </property>
     <property name="plainText">
      <string>-----
</string>
     </property>
    </widget>
    <widget class="QPushButton" name="logClearButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 127);
color: rgb(255, 255, 255);
border-width: 0px;
</string>
     </property>
     <property name="text">
      <string>CLEAR</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="logEnableCheck">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>11</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>ENABLE FROM CAN</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="debugText">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>340</y>
       <width>511</width>
       <height>211</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
</string>
     </property>
     <property name="plainText">
      <string>-----
</string>
     </property>
    </widget>
    <widget class="QPushButton" name="debugClearButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>300</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 127);
color: rgb(255, 255, 255);
border-width: 0px;
</string>
     </property>
     <property name="text">
      <string>CLEAR</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>320</y>
       <width>231</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
color: rgb(255, 255, 255);
border-width: 0px;
</string>
     </property>
     <property name="text">
      <string>DEBUG TEXT</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QCheckBox" name="debugEnable">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>300</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>ENABLE</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="logEnableEthCheck">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>10</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>ENABLE FROM ETH</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
